tests/lib/ansible_helper.py

```diff
@@ -147,8 +148,18 @@ def create_inventory(self, hardware):
         with open(nodes_inv_path, 'w') as inv_file:
             yaml.dump(inv, inv_file)
 
+        logger.info('Inventory path: {}'.format(inventory_dir))
         return inventory_dir
 
+    def run_play_raw(self, playbook):
+        path = os.path.join('tests/assets/ansible', playbook)
+        logger.info(f'Running playbook {path}')
+        try:
+            subprocess.run(['ansible-playbook', '-i', self.inventory_dir,
+                            path])
+        except subprocess.CalledProcessError:
+            logger.error('An error occured executing Ansible playbook')
```

Comment:
Why don't we do something here? exit or handle the error? Now we just continue and we'll run into a follow up error that will be hard to debug...